.github/ISSUE_TEMPLATE/04_thanks.md

```diff
@@ -0,0 +1,23 @@
+# ❤️ I'm using Nock
+
+---
+name: 🤗 Support us
+about: If you want support our efforts in maintaining Nock
+
+---
+
+Click "Preview" for a nicer view!
```

Comment:
I wonder if we should let people create issues for this template. We can add checkboxes for the things they can do, e.g.

```
- [ ] Star the repository ⭐️
- [ ] Help out with issues
- [ ] ...
```

We could let people say a sentence or two, it will be nice to hear for the maintainers. We could also ask them in what projects they use it by what companies, if they are okay sharing that information

Reply:
Hi! I've updated the docs. Let me know if I missed anything on this file.